doc/sphinx/Pacemaker_Explained/operations.rst

```diff
@@ -38,113 +38,154 @@ two operations for the same resource with the same name and interval.
 Operation Properties
 ####################
 
-Operation properties may be specified directly in the ``op`` element as
-XML attributes, or in a separate ``meta_attributes`` block as ``nvpair`` elements.
-XML attributes take precedence over ``nvpair`` elements if both are specified.
+An operation property may be specified in any of these ways, from highest
+precedence to lowest:
 
-.. table:: **Properties of an Operation**
+* directly in the ``op`` element as an XML attribute
+* in an ``nvpair`` element within a ``meta_attributes`` element within the
+  ``op`` element
+* in an ``nvpair`` element within a ``meta_attributes`` element within
+  :ref:`operation defaults <s-operation-defaults>`
+
+If not specified, the default from the table below is used. The ``id``,
+``name``, ``interval``, and ``role`` properties may only be specified in the
+``op`` element.
```

Comment:
Somewhat contradicts the first sentence of this section, but this is probably less awkward than saying "With a few exceptions listed below, ...".

One option would be to start with something like "The ``id``, ... properties must be specified directly in the ``op`` element as an XML attribute. Other properties may be specified in any of the following ways...:"

I don't feel strongly.

---

The name `PCMK_META_INTERVAL` might have been a mistake, oh well... it does get stored in the metadata at some point though.

Reply:
> Somewhat contradicts the first sentence of this section, but this is probably less awkward than saying "With a few exceptions listed below, ...".
> 
> One option would be to start with something like "The `id`, ... properties must be specified directly in the `op` element as an XML attribute. Other properties may be specified in any of the following ways...:"
> 
> I don't feel strongly.

makes sense

> 
> The name `PCMK_META_INTERVAL` might have been a mistake, oh well... it does get stored in the metadata at some point though.

Of course it's fine to have both META and XA names